shared_libraries/orca_shared/recovery/test/unit_tests/test_shared_recovery.py

```diff
@@ -94,6 +94,12 @@ def test_post_entry_to_fifo_queue_no_errors(self):
                         }
                     ]
                 )
+                # Verifies that request_method is set properly (new_job or update_file)
+                self.assertTrue(
+                    request_method.value == "new_job"
```

Comment:
Shouldn't this be specific to either `new_job` or `update_file`?

Reply:
So this will check if request_method is set as `new_job` or `update_file`, as long as request_method has one of those set the test will pass, but if request_method is set to something else like `delete_file` the test will fail and output a message with what the incorrect value is.

Reply:
I think we should also have a test that fails in case of `request_method` equals to something else other than `new_job` or `update_file`

Reply:
This test will fail if `request_method` is set to something else other than `new_job` or `update_file`. Tested by adding new request methods and `request_method` was set to `delete_file` and the test failed:
`AssertionError: False is not true : Incorrect Request Method: delete_file`

Bamboo output: https://ci.earthdata.nasa.gov/browse/ORCA-OI405-JOB1-5/log#:~:text=AssertionError%3A%20False%20is%20not%20true%20%3A%20Incorrect%20Request%20Method%3A%20delete_file

So if `request_method` is set to `new_job` or `update_file` the test will pass, and if it's equal to anything else the test will fail.

Reply:
Understood but lets add that in a new test which uses self.assertRaises or something similar to verify that it is indeed failing.

Something like: 
 test_request_method_raises_error(self)
...........